src/api-client/mock/index.ts

```diff
@@ -123,6 +124,38 @@ export default {
 
     return returnResult(token, userName, fetch(user)) as Promise<InternalActor>;
   },
+  async updateProfile(
+    token: string,
+    userName: string,
+    updateProfile: UpdateProfile
+  ): Promise<InternalActor> {
+    console.info(
+      `updateProfile => token: ${token}, userName: ${userName}, currentPassword: ${updateProfile.currentPassword}, newPassword: ${updateProfile?.newPassword}, email: ${updateProfile?.email}`
```

Comment:
do we really want to log passwords in clear text? Bit risky, wdyt?

Reply:
I log it in the mock service for testing. Not used in production